cirq/contrib/quirk/url_to_circuit.py

```diff
@@ -36,7 +38,8 @@ def quirk_url_to_circuit(
         *,
         qubits: Optional[Sequence['cirq.Qid']] = None,
         extra_cell_makers: Union[Dict[str, 'cirq.Gate'], Iterable[
-            'cirq.contrib.quirk.cells.CellMaker']] = ()) -> 'cirq.Circuit':
+            'cirq.contrib.quirk.cells.CellMaker']] = (),
+        max_operation_count: Optional[int] = None) -> 'cirq.Circuit':
```

Comment:
This is vulnerable by default. At minimum this should default to some safe and practical value, perhaps 10,000.

Even better if we make this non-optional, because `None` provides an easy, but unsafe path when someone bumps into the limit. If this is non-optional, the client of this code will have to come up with a better bound which is where the library should steer them.

Reply:
Changed.